package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "tribe-common",
-  "version": "4.14.19",
+  "version": "4.14.18",
```

Comment:
This change smells fishy.

Reply:
4.14.18 was macaron - it got nixed, so I had to back up the version #
See: https://docs.google.com/spreadsheets/d/1m5xmULGTpqRAQzMG-pVT74Ub9AnSsOT87yGFz7toLTE/edit#gid=1365577016
`4.14.18` is correct

Reply:
Now I need to go check mooncake's version numbers